.github/workflows/inspektor-gadget.yml

```diff
@@ -1603,7 +1603,7 @@ jobs:
         if [ ${{ matrix.arch }} = 'arm64' ]; then
           node_type='a1.xlarge'
         fi
-        eksctl create cluster --name ${{ env.CLUSTER_NAME }} --tags ig-ci=true --node-type $node_type
+        eksctl create cluster --name ${{ env.CLUSTER_NAME }} --tags ig-ci=true,ig-ci-timestamp=$(date +%s) --node-type $node_type
```

Comment:
It would be more readable with something like:
```
$ date --iso-8601=seconds
2024-04-03T15:47:20+02:00
```

Reply:
I chose this representation so I could make the condition easier:
`if [ $cf_stack_timestamp -gt $(date -d "3 hours ago" +%s) ]; then`

Of course I am no bash expert :D Is there a easy way with your more readable representation?